Formula/lib/libdpp.rb

```diff
@@ -0,0 +1,47 @@
+class Libdpp < Formula
+  desc "C++ Discord API Bot Library"
+  homepage "https://github.com/brainboxdotcc/DPP"
+  url "https://github.com/brainboxdotcc/DPP/releases/download/v10.0.28/DPP-10.0.28.tar.gz"
+  sha256 "aa0c16a1583f649f28ec7739c941e9f2bf9c891c0b87ef8278420618f8bacd46"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on xcode: ["12.0", :build]
+  depends_on "libsodium"
+  depends_on "openssl@3"
+  depends_on "opus"
+  depends_on "pkg-config"
+
+  uses_from_macos "zip"
```

Comment:
Misremembered the name, it needs to be:
```suggestion
  uses_from_macos "zlib"
```

Reply:
No worries, I've corrected it to that now!